src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] inputArray = readFromFile(fromFileName).split(",");
+        String calculateResult = editFile(inputArray);
```

Comment:
This method name is not very helpful
1. First of all, you're not really editing any files - you read and process its content.
2. Name is too implicit - doesn't really tell anything what happens inside. "processTransaction()" or "calculateTransactionResults()" would make more sense.